app/models/lettings_log.rb

```diff
@@ -14,13 +14,15 @@ def validate(record)
   end
 end
 
-class CaseLog < ApplicationRecord
+class LettingsLog < ApplicationRecord
+  self.table_name = "lettings_logs"
```

Comment:
Why is this needed? Isn't this the default?

Reply:
If you're referring to the `self.table_name` line that was in the branch when I picked it up so I just updated it. This is still very much a WIP branch so I'll remove this line by the time it gets to review 😄